atomrdf/structure.py

```diff
@@ -298,7 +316,7 @@ def _make_dislocation(
         C,
         burgers_vector,
         dislocation_line,
-        slip_vector,
```

Comment:
Slip_vector is substituted by Slip_plane? I had understood that what was needed was the normal vector to the slip plane? Also in the doc string it's still slip_vector

Reply:
substituted with `slip_system`, which is `[slip_direction, slip_plane]`. `slip_plane` is array of length three which should be equal to normal vector. Removed `slip_vector` from docstrings.

Reply:
Fixed the issue where the code did not match with the ontology triples. Now slip plane -> normal vector -> with the given input.  The function argument is still called `slip_plane` to keep it intuitive.